apps/ts.c

```diff
@@ -887,8 +887,12 @@ static TS_VERIFY_CTX *create_verify_ctx(const char *data, const char *digest,
             goto err;
         f = TS_VFY_VERSION | TS_VFY_SIGNER;
         if (data != NULL) {
+            BIO *out = NULL;
```

Comment:
Please add a blank line after the declaration and before the following code to be consistent with our preferred style.